### PR TITLE
Fix Notes and Examples in Push-Location.md

### PR DESCRIPTION
**Pull request by sethvs**
# PR Summary

- Fix `~` symbol description
- Fix PlatyPS notes rendering

PlatyPS renders notes like this:
```
> [!NOTE] > `Push-Location` cannot add a location to the unnamed default stack unless it is the current > location stack.
```
so it doesn't remove `>` symbols. We don't need to insert the `>` symbol before every line in *.md files, only the first one, as long as there are no empty lines.

## PR Context

Select the area of the Table of Contents containing the documents being changed.

**Conceptual content**
- [ ] Overview and Install
- [ ] Learning PowerShell
  - [ ] PowerShell 101
  - [ ] Deep dives
  - [ ] Remoting
- [ ] Release notes (What's New)
- [ ] Windows PowerShell
  - WMF, ISE, release notes, etc.
- [ ] DSC articles
- [ ] Community resources
- [ ] Sample scripts
- [ ] Gallery articles
- [ ] Scripting and development
  - [ ] Legacy SDK

**Cmdlet reference & about_ topics**
- [x] Version 7.1 preview content
- [x] Version 7.0 content
- [x] Version 6 content
- [x] Version 5.1 content

## PR Checklist

- [x] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [ ] Includes content related to issues and PRs - see [Closing issues using keywords][key].
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[WIP]` to the beginning of the
    title and remove the prefix when the PR is ready.

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

File: reference/5.1/Microsoft.PowerShell.Management/Push-Location.md [hunk 1]
```diff
@@ -81,8 +81,8 @@ PS C:\>
 ```
 
 The first command pushes the current location onto a new stack named Stack2, and then changes the
-current location to the home directory, which is represented in the command by the tilde symbol (~)
-(same as `$env:USERPROFILE` or `$HOME`).
+current location to the home directory, represented in the command by the tilde symbol (~),
```
Comment (sdwheeler) [inline reference/5.1/Microsoft.PowerShell.Management/Push-Location.md:84]:
```suggestion
current location to the home directory, represented in the command by the tilde symbol (`~`),
```

File: reference/6/Microsoft.PowerShell.Management/Push-Location.md [hunk 2]
```diff
@@ -81,8 +81,8 @@ PS C:\>
 ```
 
 The first command pushes the current location onto a new stack named Stack2, and then changes the
-current location to the home directory, which is represented in the command by the tilde symbol (~)
-(same as `$env:USERPROFILE` or `$HOME`).
+current location to the home directory, represented in the command by the tilde symbol (~),
```
Comment (sdwheeler) [inline reference/6/Microsoft.PowerShell.Management/Push-Location.md:84]:
```suggestion
current location to the home directory, represented in the command by the tilde symbol (`~`),
```

File: reference/7.0/Microsoft.PowerShell.Management/Push-Location.md [hunk 3]
```diff
@@ -81,8 +81,8 @@ PS C:\>
 ```
 
 The first command pushes the current location onto a new stack named Stack2, and then changes the
-current location to the home directory, which is represented in the command by the tilde symbol (~)
-(same as `$env:USERPROFILE` or `$HOME`).
+current location to the home directory, represented in the command by the tilde symbol (~),
```
Comment (sdwheeler) [inline reference/7.0/Microsoft.PowerShell.Management/Push-Location.md:84]:
```suggestion
current location to the home directory, represented in the command by the tilde symbol (`~`),
```

File: reference/7.1/Microsoft.PowerShell.Management/Push-Location.md [hunk 4]
```diff
@@ -81,8 +81,8 @@ PS C:\>
 ```
 
 The first command pushes the current location onto a new stack named Stack2, and then changes the
-current location to the home directory, which is represented in the command by the tilde symbol (~)
-(same as `$env:USERPROFILE` or `$HOME`).
+current location to the home directory, represented in the command by the tilde symbol (~),
```
Comment (sdwheeler) [inline reference/7.1/Microsoft.PowerShell.Management/Push-Location.md:84]:
```suggestion
current location to the home directory, represented in the command by the tilde symbol (`~`),
```

File: reference/5.1/Microsoft.PowerShell.Management/Push-Location.md [hunk 5]
```diff
@@ -157,8 +157,8 @@ To make a location stack the current location stack, use the StackName parameter
 `Set-Location` cmdlet. For more information about location stacks, see the [Notes](#notes).
 
 > [!NOTE]
-> `Push-Location` cannot add a location to the unnamed default stack unless it is the current
-> location stack.
+`Push-Location` cannot add a location to the unnamed default stack unless it is the current
+location stack.
```
Comment (sdwheeler) [inline reference/5.1/Microsoft.PowerShell.Management/Push-Location.md:161]: This is a bug in PlatyPS that we need to fix. I don't want to change how we author NOTE blocks when we are in the process of fixing PlatyPS. Please revert this change.

File: reference/6/Microsoft.PowerShell.Management/Push-Location.md [hunk 6]
```diff
@@ -157,8 +157,8 @@ To make a location stack the current location stack, use the StackName parameter
 `Set-Location` cmdlet. For more information about location stacks, see the [Notes](#notes).
 
 > [!NOTE]
-> `Push-Location` cannot add a location to the unnamed default stack unless it is the current
-> location stack.
+`Push-Location` cannot add a location to the unnamed default stack unless it is the current
+location stack.
```
Comment (sdwheeler) [inline reference/6/Microsoft.PowerShell.Management/Push-Location.md:161]: Please revert this change.

File: reference/7.0/Microsoft.PowerShell.Management/Push-Location.md [hunk 7]
```diff
@@ -157,8 +157,8 @@ To make a location stack the current location stack, use the StackName parameter
 `Set-Location` cmdlet. For more information about location stacks, see the [Notes](#notes).
 
 > [!NOTE]
-> `Push-Location` cannot add a location to the unnamed default stack unless it is the current
-> location stack.
+`Push-Location` cannot add a location to the unnamed default stack unless it is the current
+location stack.
```
Comment (sdwheeler) [inline reference/7.0/Microsoft.PowerShell.Management/Push-Location.md:161]: Please revert this change.

File: reference/7.1/Microsoft.PowerShell.Management/Push-Location.md [hunk 8]
```diff
@@ -157,8 +157,8 @@ To make a location stack the current location stack, use the StackName parameter
 `Set-Location` cmdlet. For more information about location stacks, see the [Notes](#notes).
 
 > [!NOTE]
-> `Push-Location` cannot add a location to the unnamed default stack unless it is the current
-> location stack.
+`Push-Location` cannot add a location to the unnamed default stack unless it is the current
+location stack.
```
Comment (sdwheeler) [inline reference/7.1/Microsoft.PowerShell.Management/Push-Location.md:161]: Please revert this change.

File: reference/5.1/Microsoft.PowerShell.Management/Push-Location.md (hunk 5 above)
Comment (sethvs) [reply reference/5.1/Microsoft.PowerShell.Management/Push-Location.md]: Agree.
Good to hear that PlatyPS in being fixed.
